src/chains-config/metadata-consts/polkadot-consts.ts

```diff
@@ -0,0 +1,68 @@
+import { Definition } from '../../types/chains-config';
+
+/**
+ * All Runtimes before v27
+ */
+const extrinsicBaseWeight = 125000000;
+
+/**
+ * Runtimes after v26
+ */
+const blockWeights = {
```

Comment:
I am wondering if both kusama and polkadot should point at the same `blockWeights` and `extrinsicBaseWeights` object? Maybe we can make a shared file within this folder and import into these chain specific files?

Reply:
Hmm, can you elaborate on why the two are unlikely to use different weights?

Reply:
At the moment they use the same weights, so to DRY things we can just have one definition. 

In the future it is entirely possible that the two will diverge, but we will always need these definitions for older runtimes and might as well just define it once. So we can move the current `extrinsicBaseWeight` and `blockWeights` def to a common file (perhaps with more descriptive names linking them to the chain and runtime version) and then re-import them into both `polkadotConsts.ts` and `kusamaConsts.ts` and line them up to the appropriate runtime. If/when there is an update that diverges we would just add another def specific to that chain/runtime.